games-board/cockatrice/cockatrice-20230914.ebuild

```
@@ -0,0 +1,77 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit xdg cmake optfeature
+
+MY_PV="${PV:0:4}-${PV:4:2}-${PV:6:2}-Release-2.9.0"
+
+DESCRIPTION="Open-source multiplatform software for playing card games over a network"
+HOMEPAGE="https://github.com/Cockatrice/Cockatrice"
+SRC_URI="https://github.com/Cockatrice/Cockatrice/archive/refs/tags/${MY_PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/Cockatrice-${MY_PV}"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+client +oracle test server"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-libs/protobuf
```

```suggestion
	dev-libs/protobuf:=
```

cockatrice needs to rebuilt on protobuf soname upgrades